src/components/FeedbackForm.tsx

```diff
@@ -0,0 +1,97 @@
+import { IonButton, IonInput, IonItem, IonLabel, IonModal, IonTextarea } from '@ionic/react';
+import React, { useState } from 'react';
+import './Components.css';
+import emailjs from 'emailjs-com';
+import { decrypt } from '../utils/SecurityUtils';
+
+/**
+ * Feedback Screen.
+ *
+ * Uses EmailJS as a service to send emails.
+ * This class will send actual emails when people have feedback.
+ */
+const FeedbackScreen: React.FC = () => {
+  const ENCRYPTED_USER_ID = 'U2FsdGVkX19ijiVnA6XtmJ4wa/RL9NngWwn4uKfq6gO+4ZI/V1F/RCYL4REfk0tZ';
```

Comment:
This is an "encrypted" code of the actual `user_id`, but I kind of defeat the purpose of doing that by making the private key right here also. This is more of a formality and a psychological barrier to trolls than a real security wall. 

Eventually we'd probably want to store secrets in environment variables or something, if anyone knows how then I can add it to this PR, otherwise it might just be easier to keep it like this so it doesn't mess up our local development.